melissa/profile_populator.py

```diff
@@ -94,7 +94,7 @@ def empty(variable):
  or <ENTER> for a search based on the name\
  of the city you live in: ')
     if empty(city_code):
-        city_list = pywapi.get_loc_id_from_weather_com(city_name)
+        city_list = pywapi.get_loc_id_from_weather_com(unicode(city_name))
```

Comment:
the build is succesful and the test for `profile_populator` can be found by pytest.

simple merge test build https://travis-ci.org/rachmadaniHaryono/Melissa-Core/builds/167359102